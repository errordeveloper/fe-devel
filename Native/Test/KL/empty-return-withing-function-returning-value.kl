/*
 *  Copyright 2010-2012 Fabric Engine Inc. All rights reserved.
 */

function Integer foo()
{
  return;
}
