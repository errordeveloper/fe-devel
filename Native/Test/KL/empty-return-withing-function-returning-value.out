empty-return-withing-function-returning-value.kl:7:3: error: function must return a value
empty-return-withing-function-returning-value.kl:5:1: error: not all paths return a value
